scripts/install

```diff
@@ -13,82 +36,147 @@ set -eu
+# SPECIFIC SETUP
+#=================================================
+# CUSTOMIZE DOKUWIKI
+#=================================================
+
+# Set the "language"
+ynh_replace_string "__YNH_LANGUAGE__" "$language" "../conf/local.php"
+
+# Set the "admin" user
+ynh_replace_string "__YNH_ADMIN_USER__" "$admin" "../conf/local.php"
+
+# Copy Yunohost specific configuration
+cp ../conf/local.php $final_path/conf
+
+
+# Restrict user rights by enforcing "read-only" mode for all users
+# See https://www.dokuwiki.org/acl#background_info
+# Default is "8"
+cp ../conf/acl.auth.php $final_path/conf
+
+#=================================================
+# STORE THE CHECKSUM OF THE CONFIG FILE
+#=================================================
+
+# Calculate and store the config file checksum into the app settings
+#ynh_store_file_checksum "$final_path/conf/local.php"
+#=================================================
 
-# Get source
-SETUP_SOURCE
+#=================================================
+# SETUP SYSTEMD
+#=================================================
 
-sudo cp ../conf/dokuwiki.php $final_path/conf
-sudo cp ../conf/acl.auth.php $final_path/conf
+# Create a dedicated systemd config
+#ynh_add_systemd_config
+
+#=================================================
+# GENERIC FINALIZATION
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
 
 # Files owned by dokuwiki can just read
-sudo chown -R root: $final_path
+chown -R root: $final_path
 
 # except for conf, data, some data subfolders, and lib/plugin, where www-data must have write permissions
-sudo chown -R $app:root $final_path/{conf,data,data/attic,data/cache,data/index,data/locks,data/media*,data/meta,data/pages,data/tmp,lib/plugins,lib/tpl}
-sudo chmod -R 700 $final_path/conf
-sudo chmod -R 700 $final_path/data
-sudo chmod -R 755 $final_path/lib/plugins
-sudo chmod 755 $final_path/lib/tpl/{dokuwiki,dokuwiki/images}
-
-# Modify Nginx configuration file and copy it to Nginx conf directory
-sudo cp ../conf/nginx.conf /etc/nginx/conf.d/$domain.d/$app.conf
-sudo sed -i "s@__PATHTOCHANGE__@$path@g" /etc/nginx/conf.d/$domain.d/$app.conf
-sudo sed -i "s@__FINALPATH__@$final_path@g" /etc/nginx/conf.d/$domain.d/$app.conf
-sudo sed -i "s@__NAMETOCHANGE__@$app@g" /etc/nginx/conf.d/$domain.d/$app.conf
-
-if [ "$is_public" = "Yes" ];
-then
-	sudo sed -i "s@#--PRIVATE--@@g" /etc/nginx/conf.d/$domain.d/$app.conf
-fi
+chown -R $app:root $final_path/{conf,data,data/attic,data/cache,data/index,data/locks,data/media*,data/meta,data/pages,data/tmp,lib/plugins,lib/tpl}
+chmod -R 700 $final_path/conf
+chmod -R 700 $final_path/data
+chmod -R 755 $final_path/lib/plugins
+chmod 755 $final_path/lib/tpl/{dokuwiki,dokuwiki/images}
+
+#=================================================
+# SETUP LOGROTATE
+#=================================================
+
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
```

Comment:
Is there a specific log for dokuwiki ?
I thought dokuwiki was a simple php app.

Reply:
Ok, so I don't think that dokuwiki as a specific log, usually php apps log in nginx.

Reply:
Ok, will remove it